dependencies.gradle

```diff
@@ -29,6 +29,7 @@ def vanniktechEmoji = "0.8.0"
 def mockk = "1.12.1"
 def espresso = "3.4.0"
 def androidxTest = "1.4.0"
+def androidxOrchestrator = "1.4.1"
```

Comment:
Why not just update the value of `androidxTest` above to `1.4.1`?

Reply:
As you can see below only `androidxOrchestrator` is yet it `1.4.1`
<img width="413" alt="Screenshot 2022-01-07 at 17 53 50" src="https://user-images.githubusercontent.com/60798129/148569875-14d1ee90-089c-4ab3-ac67-cee429d03817.png">

Reply:
You are right, I read a bit fast the title at https://developer.android.com/jetpack/androidx/releases/test#core_141_2

Also confirmed that this should not be a problem to use various version by the sample at https://developer.android.com/jetpack/androidx/releases/test#declaring_dependencies